ui/apps/platform/src/Containers/Compliance/Dashboard/ComplianceDashboardPage.tsx

```diff
@@ -47,6 +51,10 @@ const queriesToRefetchOnPollingComplete = [
 ];
 
 function ComplianceDashboardPage(): ReactElement {
+    const [disclaimerAccepted, setDisclaimerAccepted] = useBooleanLocalStorage(
```

Comment:
What do you think are pro and con of `isDisclaimerAccepted` for the name?

Reply:
Consistency is key. I'd say the only con is it's 2 characters longer 😆 

Will update.